cmake/TPLs/FindTPLAMD.cmake

```diff
@@ -19,6 +19,11 @@ if (AMD_ALLOW_PREFIND)
 endif()
 
 if (NOT TARGET AMD::all_libs)
+
+  if ( NOT AMD_INCLUDE_DIRS )
+    set(AMD_INCLUDE_DIRS "/usr/include/suitesparse")
+  endif ()
```

Comment:
It might be better to implement this as:

```
set(AMD_INCLUDE_DIRS_DEFAULT "/usr/include/suitesparse")
set(AMD_INCLUDE_DIRS "${AMD_INCLUDE_DIRS_DEFAULT}" CACHE PATH
  "Default path to find AMD include files")
```

That way, the user will see this default in the `CMakeCache.txt` file and will know to change this if they want.